ocs_ci/helpers/dr_helpers.py

```
@@ -3062,3 +3065,50 @@ def validate_protection_label(kind, namespace, protection_name=None):
+    odf_cli_runner = odf_cli_setup_helper()
+
+    output_dir = os.path.join(
+        os.path.expanduser(config.RUN["log_dir"]),
+        f"dr_test_{action}_{config.RUN['run_id']}",
+        config.ENV_DATA["cluster_name"],
+        "out",
+        "test-subscr",
+    )
+    create_directory_path(output_dir)
+
+    logger.info(f"Running ODF DR test {action}, output will be stored at: {output_dir}")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)